fcomm/src/lib.rs

```diff
@@ -1182,136 +1238,81 @@ mod test {
             println!("Commitment: {:?}", commitment);
         }
     }
+
     proptest! {
```

Comment:
I'm actually tempted to just remove those: once we have tested the serde round_trip, all that's left is checking Rust can open a file, write to it, and then read the same bytes from the file system. That seems safe. Thoughts, @samuelburnham ?